azure-stack/hci/concepts/firewall-requirements.md

```diff
@@ -58,7 +60,7 @@ The following table provides a list of recommended firewall URLs. If your outbou
 
 Depending on additional Azure services you enable on HCI, you may need to make additional firewall configuration changes. Refer to the following links for information on firewall requirements for each Azure service:
 
-- [AKS on Azure Stack HCI](/azure-stack/aks-hci/system-requirements?tabs=allow-table#aks-on-azure-stack-hci-requirements)
+- [AKS on Azure Stack HCI](/azure/aks/hybrid/aks-hci-network-system-requirements#firewall-url-exceptions)
```

Comment:
Is one of these links the ARB?

Reply:
It added a new link to https://learn.microsoft.com/en-us/azure/aks/hybrid/aks-hci-network-system-requirements#firewall-url-exceptions , and ask customer to follow those addresses making sure AKS Arc works.  The list has been validated by engineering team should include links to ARB.